jdbc/src/java/org/apache/hive/jdbc/Utils.java

```diff
@@ -890,4 +885,18 @@ public static String getPassword(Map<String, String> confMap, String key) {
     }
     return password;
   }
+
+  /**
+   * Method to get the default port based on the transport mode
+   * @param connectionParams JDBC connection parameters
+   * @return defaultPort
+   */
+  public static String getDefaultPort(JdbcConnectionParams connectionParams) {
```

Comment:
Please address the two checkstyle issues.
https://sonarcloud.io/project/issues?id=apache_hive&pullRequest=6137&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true